src/faq.html

```diff
@@ -0,0 +1,109 @@
+---
+layout: base
+description: FAQ | SkyConnect
+faqs:
+  - title: >
+      Can I run 2 sticks with ZHA simultaneously?
+    content: >
+      <p>No. ZHA (Zigbee Home Automation) is single instance only. ZHA cannot control 2 radios at once.</p>
+  - title: >
+      Can I run 2 sticks with different Zigbee solutions simultaneously?
+    content: >
+      <p>
+        Yes. For example, you can run a stick with deCONZ or Zigbee2MQTT alongside SkyConnect with ZHA.
+      </p>
+  - title: >
+      Can I migrate from a deCONZ or Zigbee2MQTT stick to SkyConnect and keep using the old stick?
+    content: >
+      <p>No. During migration, the IEEE address of the SkyConnect is replaced by the IEEE address of the old stick. This means that your old stick and your new stick will have the same IEEE address. This is required for the migration process to complete successfully. From this point on-wards you should not operate the old stick in the same area. If you do not migrate the IEEE address from your old stick, you'll have to re-pair many of your devices in order to keep them working.</p>
```

Comment:
I bet people don't read the full title, so we should make them stumble on the first words.

```suggestion
      <p>No, you can't continue to use both. During migration, the IEEE address of the SkyConnect is replaced by the IEEE address of the old stick. This means that your old stick and your new stick will have the same IEEE address. This is required for the migration process to complete successfully. From this point on-wards you should not operate the old stick in the same area. If you do not migrate the IEEE address from your old stick, you'll have to re-pair many of your devices in order to keep them working.</p>
```